src/components/Search/Search.js

```diff
@@ -0,0 +1,105 @@
+import React, { useMemo, useState } from 'react';
+import Autosuggest from 'react-autosuggest';
```

Comment:
Have you seen our Autocomplete component in `react-spatial` ?  I'm not against using a better autocomplete component but we  fix a lot of behavior stuff (css, keyboard navigation) and bugs that I don't want to fix them again :)

Reply:
Mario and me discussed this and we would like to remove non-spatial components from react-spatial. In my opinion we should move them into a generic UI component library or use third-party component libraries. This was my attempt on the autocomplete library. But we should discuss this topic together and decide sooner than later how to proceed.

Reply:
I agree we should remove these components from react-spatial.